.github/workflows/ci.yaml

```diff
@@ -54,6 +54,10 @@ jobs:
         run: yarn lint
         if: success()
 
+      - name: Run code-server unit tests
+        run: yarn test:unit
+        if: success()
+
```

Comment:
I'm guessing this was removed by accident but it should be here and we need it before uploading coverage to Codecov (it outputs coverage information).